### ISSUE TITLE
[Feature Request]: Wither protection in claimed chunks

### ISSUE THREAD

**Issue by winnetrie83** [FTBTeam/FTB-Mods-Issues#1144]
### Mod

FTB Chunks

### Description

I hereby request a feature for FTBchunks having an option to have protection against the wither.
Explosion are protected, but the wither can still break blocks.
While gamerule mobgriefing can be toggled, it does also interfere with other things that are considered mobgriefing. a villager picking up an item for example.
So an option to toggle wither protection would be very helpfull.

### Describe the solution you'd like

Adding an option to toggle wither protection. Same as explosion protection right now.

### Describe alternatives you've considered

mobgriefing gamerule (not preferred)

**Comment by desht** [comment]
This would trivialize wither boss fights, though?

**Comment by winnetrie83** [comment]
Yes sure a server config would be great.
It's just an idea  a request. It can be denied ofcourse. 
Besides that, i already found a mod that has this feature, more mobgriefing options. So no pressure. 😅😁

**Comment by MichaelHillcox** [comment]
> It's a possbility, but would need to controlled by a server config setting (off by default). Allowing any player to stop wither damage by default would just be far too OP.

Yeah, this is essentially what we discussed in Discord. It would need to be an opt in system and server controlled

**Comment by desht** [comment]
Given that other mods can handle this, I'm closing this issue.
